scripts/createSymLinksOCPKG.sh

```diff
@@ -95,17 +95,17 @@ if ! [ -z $GITROOT ]; then
 		echo "Creating folders for symbolic links..."
 
 		# Let's create the folders first
-		mkdir -p bin/opencog/server
+		mkdir -p bin/opencog/cogserver/server
```

Comment:
"bin" ??? do you mean "build"?   There is no "bin" directory, there is a build directory, where the source code gets built.